doc/WebOnChain.plantuml

```diff
@@ -0,0 +1,186 @@
+@startuml
+participant "End user" as user
+participant "Browser" as ui
+participant "Thyra server" as thyra
+participant "Hosting websites" as web
+participant "Wallet" as w
+participant "DNS" as dns
+participant "Event Manager" as em
+participant "Storage" as storage
+participant "Node (API)" as node
+
+user -> ui: click on install 'Web on-chain' plug-in
+ui -> thyra: GET /plugin/webOnChain
+thyra -> web: GET /plugin/webOnChain
+web -> thyra: OK response
+thyra -> ui: display landing page WebOnChain.massa
```

Comment:
How does it work ?
- When you create the webOnChain plugin, you natively have the webonchain.massa domain ?
- You need to by it manually ?
- Should we reserved/block `plugin` resource at the DNS level ?

Final note: URL are case sensitive. Do we plan to allow webOnChain, WebOnChain, webonChain, webonchain to be valid values and all pointing to different urls ?